test/conformance/chainsaw/cleanup/clusterpolicy/default-apicall/chainsaw-test.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: default-apicall
+spec:
+  steps:
+  - name: step-01
+    try:
+    - apply:
+        file: rbac.yaml
```

Comment:
We should simplify this by using a standard cluster policy, not a cleanup policy. This test does not test anything specific to cleanup.

Reply:
I will change it to cluster policy